Tests/MongoSwiftSyncTests/SyncTestUtils.swift

```diff
@@ -115,49 +115,47 @@ extension MongoClient {
 /// Captures any command monitoring events filtered by type and name that are emitted during the execution of the
 /// provided closure. Only events emitted by the provided client will be captured.
 internal func captureCommandEvents(
```

Comment:
These methods aren't as useful anymore, but I kept them in to try to reduce the diff on this change as much as possible.

Reply:
should we open a ticket about converting to the new way of doing things at some point in the future?